src/libraries/System.ComponentModel.TypeConverter/tests/ContainerTests.cs

```diff
@@ -322,7 +322,7 @@ public void Add2_Name_Duplicate()
             Assert.Equal(typeof(ArgumentException), ex.GetType());
             Assert.Null(ex.InnerException);
             Assert.NotNull(ex.Message);
-            Assert.True(ex.Message.IndexOf("'dup'") != -1);
+            Assert.True(ex.Message.IndexOf("'dup'") != -1 || ex.Message.IndexOf(" dup") != -1);
```

Comment:
What's the reason for omitting the apostrophes in the fallback string?

Reply:
Ah ok, this is the exception message. Makes sense.

Reply:
The format string from resources is: `Duplicate component name '{0}' ...``

The synthetized format string that you get with UseSystemResourceKeys is `DuplicateComponentName {0}`. Notice that the apostrophes are missing.

Reply:
OK, that makes sense. But I guess I don't understand how this test is getting run with `UseSystemResourceKeys=true`.

Reply:
I assume that @tarekgh has done a one-off manual run with `UseSystemResourceKeys=true`. I do not think we have it automated.

Reply:
That is right, I have run the test with and without enabling UseSystemResourceKeys and ensured it is passing in both ways. Enabling `UseSystemResourceKeys` in general should be done separately from this PR. I'll open issue to track that.

Reply:
I have logged the issue https://github.com/dotnet/runtime/issues/68707 to track enabling `UseSystemResourceKeys` on libraries tests.